### PR TITLE
Release 0.9.3

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Follow https://github.com/paritytech/polkadot/pull/1443

### CODE REVIEW

File: runtime/darwinia/src/lib.rs [hunk 1]
```diff
@@ -173,13 +173,13 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
...
 	authoring_version: 0,
-	spec_version: 20,
+	spec_version: 21,
 	impl_version: 0,
 	#[cfg(not(feature = "disable-runtime-api"))]
 	apis: RUNTIME_API_VERSIONS,
 	#[cfg(feature = "disable-runtime-api")]
 	apis: sp_version::create_apis_vec![[]],
-	transaction_version: 0,
+	transaction_version: 1,
```
Comment (HackFisher) [inline runtime/darwinia/src/lib.rs:182]: When should we bump transaction_version? Any signals or convention to follow?

**Comment by HackFisher** [comment]
@AurevoirXavier Since this is an important public event for community with prediction, may wish to test again on local testnet though I'm quite sure it will work.

File: runtime/darwinia/src/lib.rs (hunk 1 above)
Comment (AurevoirXavier) [reply runtime/darwinia/src/lib.rs:182]: If the payload changed, bump it. Else, whatever.

I would like to follow polkadot here.

**Comment by AurevoirXavier** [comment]
> @AurevoirXavier Since this is an important public event for community with notice, may wish to test again on local testnet though I'm quite sure it will work.

Okay. I'm on it.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**